apps/webservice/src/app/[workspaceSlug]/(targets)/targets/TargetPageContent.tsx

```diff
@@ -27,54 +32,62 @@ export const TargetPageContent: React.FC<{
   view: schema.TargetView | null;
 }> = ({ workspace, view }) => {
   const { filter, setFilter, setView } = useTargetFilter();
+  const workspaceId = workspace.id;
+  const targetsAll = api.target.byWorkspaceId.list.useQuery({ workspaceId });
+  const targets = api.target.byWorkspaceId.list.useQuery(
+    { workspaceId, filter },
+    { placeholderData: (prev) => prev },
```

Comment:
before every time the filter would change we would show the skeletons. now, we display the previous data and have a loading indicator that the new query is loading, without removing the current filter view for the user